.github/workflows/test.yml

```diff
@@ -7,6 +7,7 @@ on:
   schedule:
     - cron: '0 22 * * 3'
   workflow_call:
+  workflow_dispatch: 
```

Comment:
> so anyone that forks the repo can run the tests locally in the fork

Could you just have this in your fork and avoid upstreaming it? 🤔

Reply:
I  needed upstream this to run the tests easier locally [here](https://github.com/ArthurLobopro/update-electron-app-custom/actions). This grants me the power to run anytime the tests, I only use Linux and my hardware can't run VMs. So the only way I can test it in other platforms is using Github Actions. If you want I can remove it, but only when the other needed changes were done